doc/admin/qserv-ingest/ingest.yaml

```diff
@@ -0,0 +1,69 @@
+version: 15
+ingest:
+    http:
+        # Optional
+        # Timeout for POST and PUT queries in seconds
+        # POST and PUT queries which might be very long like:
+        # - postint contribution
+        # - closing transaction
+        # - publishing database
+        # have no timeout at all
+        write_timeout: 1800
+        # Optional
+        # Timeout for GET queries in seconds
+        read_timeout: 10
+    metadata:
+      # Optional
+      # Allow to customize metadata URL
+      # Default to "{servers[0]}{path}"
+      url: http://dataserver/datasets/DC2/
+
+    input:
+        # List of http servers providing input dataset
+        # The ingest process will load-balance the download of input files accross
+        # these servers.
+        # Use file:// as first element in list when using local data
```

Comment:
Would using `file://` screw your previous instructions for:
```
metadata:
      # Optional
      # Allow to customize metadata URL
      # Default to "{servers[0]}{path}"
```
In this case, the metadata operations would be attempted via the file service. Or, it still would work fine? Just curious.

Reply:
This path is for retrieving data, the metadata option support the situation where the metadata file is in an other place than the data. It can be useful to test new metadata file version with existing data.

`file://` was implemented by @hsinfang and I did not test or use this feature. I do not know if it still works fine.